spec/controllers/students_controller_spec.rb

```diff
@@ -513,94 +558,58 @@ def make_request(student)
-      it 'does not return any lasids' do
-        sign_in(educator)
-        make_request
-        expect(parsed_response).to eq({ "error"=>"You don't have the correct authorization." })
-      end
-    end
-
-    context 'no educator logged in' do
-      it 'returns an error' do
-        make_request
-        expect(parsed_response).to eq({ "error"=>"You need to sign in before continuing." })
-      end
-    end
-  end
-
   describe '#student_report' do
     let(:educator) { FactoryGirl.create(:educator, :admin, school: school) }
     let(:school) { FactoryGirl.create(:school) }
     let(:student) { FactoryGirl.create(:student, :with_risk_level, school: school) }
 
-    def make_request(options = { student_id: nil, format: :pdf, from_date: '08/15/2015', to_date: '03/16/2017'})
+    def make_request(student_id, options = {})
       request.env['HTTPS'] = 'on'
       get :student_report, params: {
-        id: options[:student_id],
-        format: options[:format],
-        from_date: options[:from_date],
-        to_date: options[:to_date],
-        disable_js: true
-      }
+        id: student_id,
+        disable_js: true,
+        format: :pdf,
+        from_date: '08/15/2015',
+        to_date: '03/16/2017'
+      }.merge(options)
     end
 
     context 'when educator is not logged in' do
       it 'does not render a student report' do
-        make_request({ student_id: student.id, format: :pdf })
+        make_request(student.id)
         expect(response.status).to eq 401
       end
     end
 
     context 'when educator is logged in' do
-      before do
-        sign_in(educator)
-        make_request({ student_id: student.id, format: :pdf, from_date: '08/15/2015', to_date: '03/16/2017' })
-      end
-
       context 'educator has schoolwide access' do
 
         it 'is successful' do
+          sign_in(educator)
+          make_request(student.id)
           expect(response).to be_success
         end
 
         it 'assigns the student correctly' do
+          sign_in(educator)
+          make_request(student.id)
           expect(assigns(:student)).to eq student
         end
 
         it 'assigns the student\'s services correctly with full history' do
           old_service = FactoryGirl.create(:service, date_started: '2012-02-22', student: student, discontinued_at: '2012-05-21')
           recent_service = FactoryGirl.create(:service, date_started: '2016-01-13', student: student, discontinued_at: nil)
+          sign_in(educator)
+          make_request(student.id)
           expect(assigns(:services)).not_to include(old_service)
           expect(assigns(:services)).to include(recent_service)
         end
 
         it 'assigns the student\'s notes correctly excluding restricted notes' do
           restricted_note = FactoryGirl.create(:event_note, :restricted, student: student, educator: educator)
           note = FactoryGirl.create(:event_note, student: student, educator: educator)
+          sign_in(educator)
```

Comment:
This was failing and not sure how it ever worked - the `before` was making the request, but then inside the `it` it was adding the models needed for the test.  This makes the order explicit.